lib/manager/ansible-galaxy/util.ts

```diff
@@ -0,0 +1,6 @@
+export const newBlockRegEx = /^\s*-\s*((\w+):\s*(.*))$/;
+export const blockLineRegEx = /^\s*((\w+):\s*(.*))$/;
+export const galaxyDepRegex = /[\w-]+\.[\w-]+/;
+export const dependencyRegex = /^dependencies:/;
+export const galaxyRegEx = /^\s+(?<lookupName>[\w.]+):\s*["'](?<version>.+)["']\s*/;
+export const nameMatchRegex = /(?<source>((git\+)?(?:(git|ssh|https?):\/\/)?(.*@)?(?<hostname>[\w.-]+)(?:(:\d+)?\/|:))(?<depName>[\w./-]+)(?:\.git)?)(,(?<version>[\w.]*))?/;
```

Comment:
```suggestion
export const nameMatchRegex = /(?<source>((git\+)?(?:(git|ssh|http(s)?):\/\/)?(.*@)?(?<hostname>[\w.-]+)(?:(:\d+)?\/|:))(?<depName>[\w./-]+)(?:\.git)?)(,(?<version>[\w.]*))?/;
```
Should be `HTTP | HTTPS`

Reply:
`https?` and `http(s)?` have the same affect. The optional parameter references the last token regardless if they are a group or static char. You can test this here https://regex101.com/

This case is covered in the unit tests: 
https://github.com/renovatebot/renovate/blob/master/lib/manager/ansible-galaxy/__fixtures__/requirements01.yml#L14